content/en/platform/corda/5.0/deploying-operating/config/tls-certification.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "TLS Certification Installation on REST Worker"
+version: 'Corda 5.0'
+date: '2023-05-29'
+menu:
+  corda5:
+    identifier: corda5-tls-certification-installation
+    parent: corda5-cluster-config
+    weight: 3075
+section_menu: corda5
```

Comment:
Can you validate that this front matter is correct ?
We have both:
```
menu:
  corda5:
```
and also:
```
section_menu: corda5
```

Reply:
The front matter isn't fully understood but the front matter variables are used in other files and work correctly!